src/components/screens/surveys-screen.tsx

```diff
@@ -1,7 +1,153 @@
+import { List, ListItem, Tab, Tabs, ListItemText } from "@mui/material";
+import { Box } from "@mui/system";
+import { useState } from "react";
+import { DateTime } from "luxon";
+import strings from "../../localization/strings";
+import { SurveyScreens } from "../../types";
+
+const mockData = [
+  {
+    id: "12313",
+    kysely: "name of survey",
+    näytöt: 3,
+    julkaisuaika: DateTime.now(),
+    päättymisaika: DateTime.now(),
+    suosituinNäyttö: "location",
+    vastauksia: 234324
+  },
+  {
+    id: "12314",
+    kysely: "name of survey2",
+    näytöt: 4,
+    julkaisuaika: DateTime.now(),
+    päättymisaika: DateTime.now(),
+    suosituinNäyttö: "location 2",
+    vastauksia: 17
+  },
+];
+
+/**
+ * Interface for tab panel props
+ */
+interface TabPanelProps {
+  children: React.ReactNode;
+  value: SurveyScreens;
+  index: SurveyScreens;
```

Comment:
These could be just numbers.

Reply:
I did these as numbers at first, I thought words/ enum might be clearer?

Reply:
We discussed and decided to leave as is